localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -319,11 +332,23 @@ def return_response(self, method, path, data, headers, response):
                 stream_spec = dynamodb_get_table_stream_specification(table_name=table_name)
                 if stream_spec:
                     record['dynamodb']['StreamViewType'] = stream_spec['StreamViewType']
+
         elif action == 'BatchWriteItem':
-            records = self.prepare_batch_write_item_records(record, data)
+            records, unprocessed_items = self.prepare_batch_write_item_records(record, data)
             for record in records:
                 event_sources_or_streams_enabled = (event_sources_or_streams_enabled or
                     has_event_sources_or_streams_enabled(record['eventSourceARN'], streams_enabled_cache))
+            if response.status_code == 200 and any(unprocessed_items):
+                content = json.loads(to_str(response.content))
+                table_name = list(data['RequestItems'].keys())[0]
+                if table_name not in content['UnprocessedItems']:
+                    content['UnprocessedItems'][table_name] = []
+                for key in ['PutRequest', 'DeleteRequest']:
+                    if any(unprocessed_items[key]):
+                        content['UnprocessedItems'][table_name].append({key: unprocessed_items[key]})
+                print(content)
```

Comment:
nitpick: can we please remove the `print(..)` before merging

Reply:
FML, my bad Waldemar. Fixed